pyproject.toml

```
@@ -19,7 +19,7 @@ maintainers = [
 dependencies = [
     "ansys-api-mapdl==0.5.1",  # supports at least 2020R2 - 2022R1
     "ansys-corba; python_version < '3.9'",
-    "ansys-mapdl-reader>=0.51.7",
+    "ansys-mapdl-reader>=0.52.9",
```

```suggestion
    "ansys-mapdl-reader>=0.51.7",
```
I would still stick to this one as a minimum

---

Yep... I got confused.... you are right.